packets.go

```diff
@@ -879,6 +888,7 @@ func (stmt *mysqlStmt) writeCommandLongData(paramID int, arg []byte) error {
 		}
 
 		stmt.mc.sequence = 0
+		stmt.mc.compressionSequence = 0
```

Comment:
`mc := stmt.mc`

then use `mc` directly for both writes

Reply:
done!